pages/Signin.vue

```diff
@@ -1,70 +1,56 @@
-            <p>Email</p>
-            <input type="email" name="" placeholder="Enter your email"/>
-          </div>
 
-
-          <div class="password">
-            <p>Password</p>
-            <input type="password" name="" placeholder="***********"/>
-          </div>
-
-          <div class="remember-forgot">
-            <div class="remember-me">
-              <input type="checkbox" name="check" id="checkbox">
-              <span class="checkmark">Remember me</span>
-            </div>
-            <div class="forgot-password">
-              <p>forgot Password</p>
+<body>
+    <div class="content-box">
+        <div class="catch-line-outer">
+            <div class="catch-line-inner">
+                <p>"Welcome back! Please enter your details"</p>
             </div>
+        </div>
+        <div class="email">
+          <p>Email</p>
+          <input type="email" name="" placeholder="Enter your email"/>
+        </div>
+        <div class="password">
+          <p>Password</p>
+          <input type="password" name="" placeholder="***********"/>
+        </div>
+        <div class="remember-forgot">
+          <div class="remember-me">
+            <input type="checkbox" name="check" id="checkbox">
+            <span class="checkmark">Remember me</span>
           </div>
-          <div class="sign-up">
+          <div class="forgot-password">
+            <a href="#">Forgot password</a>
+          </div>
+        </div>
+        <div class="sign-up">
+          <a href="#">
             <button type="submit" id="Sign-up">
               <p>Sign up</p>
             </button>
-          </div>
-
-          <div class="sign-up-google">
-            <button type="submit" id="submit-google">
-              <div class="google-img">
-              </div>
-              <p>Sign up with Google</p>
-            </button>
-          </div>
-
-          <div class="optional-sign-in">
-            <p>
-              <span class="account-text">Dont have an account?</span>
-              <span class="sign-in-text">Sign up to free</span>
-            </p>
-          </div>
-      </div>
-      <div class="img-box-outer">
-          <div class="img-box-inner">
-              <img src="../assets/images/37. order-delivered-3.png" alt="" />
-          </div>
-      </div>
-  </body>
-  </html>
-
+          </a>
+        </div>
+        <div class="sign-up-google">
+          <button type="submit" id="submit-google">
+            <div class="google-img">
+            </div>
+            <a href="#">Sign up with Google</a>
+          </button>
+        </div>
+        <div class="optional-sign-in">
+          <p>
+            <span class="account-text">Dont have an account?</span>
+            <a href="#">Sign up for free</a>
+          </p>
+        </div>
+    </div>
+    <div class="img-box-outer">
+        <div class="img-box-inner">
+            <img src="../assets/images/37. order-delivered-3.png" alt="" />
+        </div>
```

Comment:
Where is this image present in your PR? I couldn't find it out.

Reply:
As while pushing the first PR the assets folder was already sent, so i thought that it might have been there.
![2023-06-12 (2)](https://github.com/bsoc-bitbyte/GetIt/assets/111559776/cdd69149-f224-462f-84e3-2b38b933db17)